infra/tf/terraform.tfstate

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
we can remove this file and the `tf.state.backup`

Reply:
will remove all extraneous files :+1:

Reply:
Reminder

Reply:
I removed with https://github.com/fleek-network/ursa/pull/81/commits/fb2940feddf3c36100b10341ace27fe7f07ac7da

But later did a `terraform plan` realized without that file, terraform cannot correctly create a plan and will attempt to create the entire network from scratch and readded with https://github.com/fleek-network/ursa/pull/81/commits/9a28c59e07427354edb855d22c88847a01623575

Reply:
removed again: https://github.com/fleek-network/ursa/pull/81/commits/bb8a4e5c689c16e6608277422a0dac68238c1a43